### ISSUE TITLE
Toolbar search has white text on white background when displayed in header

### ISSUE THREAD

**Issue by krishollenbeck** [infor-design/enterprise#6435]
**Describe the bug**
Toolbar search has white text on white background when displayed in header

**To Reproduce**
1. Edit **example-header-tab.html**. Add placeholder attribute
`<input class="searchfield " data-options="{collapsible: false}" placeholder="Search"/>`
2. Go to http://localhost:4000/components/page-patterns/example-header-tab.html
3. Notice "Search" text is not visible. White text on white background.

**Expected behavior**
Placeholder text should be visible.

**Version**
4.62.0

**Screenshots**
<img width="1533" alt="font-color-white" src="https://user-images.githubusercontent.com/1056684/168909153-52748e48-01ba-4b62-9924-42699327d57d.png">

**Platform**
 - Windows 11
 - Chrome  101.0.4951.54

**Comment by tmcconechy** [comment]
This was a purposeful change as per https://github.com/infor-design/enterprise/issues/6047

Mainly done to solve several complaints and use cases all search fields are now white. Mentioned in the change log 

```
[Searchfield/Header] Enhanced the font colors, background colors for the searchfield inside of the header & subheader. (https://github.com/infor-design/enterprise/issues/6047)
```

**Comment by krishollenbeck** [comment]
@tmcconechy So what does this mean for placeholder value when search is in header? We have toolbar in the header with search for a lot of use cases. We can override the CSS for now..

```
.header .toolbar-searchfield-wrapper.non-collapsible .searchfield::-webkit-input-placeholder {
  color: inherit;
}
```

**Comment by tmcconechy** [comment]
It was one factor in making it all consistent - it makes the text placeholder consistent on the white background. So if that rule is intended to make the color match the header color. It's possibly not needed.

If it is for something else you should still be able to use the rule as the classes didn't change as far as i observed on the PR (which i didnt do myself)?

**Comment by krishollenbeck** [comment]
I am overriding this.. 

_searchfield-toolbar.scss
```
.header .toolbar-searchfield-wrapper.non-collapsible .searchfield::-webkit-input-placeholder, .masthead .toolbar-searchfield-wrapper.non-collapsible .searchfield::-webkit-input-placeholder, .module-tabs .toolbar-searchfield-wrapper.non-collapsible .searchfield::-webkit-input-placeholder {
    color: rgba(255, 255, 255, 0.5);
}
```

With this.. Which makes the white placeholder text display as black. 
```
.header .toolbar-searchfield-wrapper.non-collapsible .searchfield::-webkit-input-placeholder {
  color: inherit;
}
```

The color is set to white when it is in this context. The  input with a white background has white placeholder text. Possibly just a missed use case? Doesn't seem like it would be intentional to put white text on a white background.

**Comment by ericangeles** [comment]
I believe it's already fixed in 4.63.0

https://4630-enterprise.demo.design.infor.com/components/page-patterns/example-header-tab.html

Try to add palceholder

**Comment by krishollenbeck** [comment]
@tmcconechy @ericangeles  I am seeing the issue on 4.62.0. I will test out 4.63.0. I'm not sure if we are limited to 4.62.0? I believe it was the minimum version required for us to move to Angular 13.

**Comment by tmcconechy** [comment]
Yeah i think that might be the issue - we fixed the placeholder further from the original change on 4.63. 

If necessary i can build out a 10.15.0 on NG 12 but the last one with NG 12 only had https://github.com/infor-design/enterprise-ng/blob/10.14.0/projects/ids-enterprise-ng/package.json#L8 in it.

13.3.0 -> had 4.62
13.4.0 -> has 4.63

**Comment by tmcconechy** [comment]
As understood from offline comments. This issue is solved in 13.4.0 - and solved otherwise by workaround without updating. Closing for now
